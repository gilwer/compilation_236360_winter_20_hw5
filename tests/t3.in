enum days {SUNDAY, MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY};

void checkDay(enum days e1) {
	if ((int) e1 == 2) {
		print("~IT'S TAAACO TUUUUUUESDAAAAY!~");
		print("One taco for me please!");
		int i = 0;
		while (i < (int) e1) {
			print("Another taco for me please!");
			i = i + 1;
		}
	}
}

void main()
{
	enum days d = TUESDAY;
	checkDay(d);
}