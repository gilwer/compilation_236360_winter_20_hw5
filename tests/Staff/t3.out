~IT'S TAAACO TUUUUUUESDAAAAY!~
One taco for me please!
Another taco for me please!
Another taco for me please!
